src/styles.css

```diff
@@ -38,7 +38,7 @@
   padding: var(--g-text-body-1-font-size);
 }
 
-.area__card > section {
+.area-card__editor > section {
```

Comment:
since you are nuking borders from html preview, can you also remove borders from other tabs.

now layout "jumping" because of that